precompiles/werc20/events.go

```diff
@@ -19,7 +19,7 @@ const (
 	// EventTypeDeposit defines the event type for the Deposit transaction.
 	EventTypeDeposit = "Deposit"
 	// EventTypeWithdraw defines the event type for the Withdraw transaction.
-	EventTypeWithdraw = "Withdraw"
+	EventTypeWithdraw = "Withdrawal"
```

Comment:
Is this the event on the WEVMOS9 contract? If so can you also change the name and comment?